src/custom/components/swap/TradeSummary.tsx

```diff
@@ -60,13 +67,13 @@ export default function TradeSummary({ trade, allowedSlippage }: { trade: TradeW
           <RowFixed>
             <TYPE.black fontSize={14} fontWeight={400} color={theme.text2}>
               {/* Liquidity Provider Fee */}
-              GP/Gas Fee
+              Protocol/Gas Fee
```

Comment:
should address this in another PR according to: https://gnosisinc.slack.com/archives/CPZA1AGKY/p1617037582045900

Reply:
Would just be `Fees`